app/views/occupation_standards/show.html.erb

```diff
@@ -114,19 +128,6 @@
                         <% else %>
                           <p class="text-white">No documents available at the moment</p>
                         <% end %>
-                        <!--
-                        <button class="flex w-full items-center space-x-4 rounded bg-slate-700 p-2 text-left text-white hover:bg-slate-600">
```

Comment:
Looking at my [task](https://app.asana.com/0/1203289004376659/1204546181205130/f) and the figma files, I assumed this button would change its label from "View Redacted Document" to "Download Working Copy", mostly because of this comment from Patrick:

"The "Download Working Copy" link is hidden on the live site at the moment but you can see it in the figma designs".

Since there isn't anything labeled "Download Working Copy" on the codebase, I assumed this button was being renamed. From your PR, it seems like this is not the case, and I should add a new button. Does it seem right to you?

Reply:
@sarahraqueld yes. There are three different documents and "Download Working Copy" is a brand-new one independent from "View Redacted Document" and "View Original Document"